packages/core/src/scss/components/foundation/forms/_forms.scss

```diff
@@ -40,6 +40,7 @@
 
     .form-field {
         position: relative;
+        margin: 0 0 spacing("half");
```

Comment:
Does it also apply to all form fields?

Reply:
Yes, that's the goal from the designs.

Reply:
Could you please provide before/after shipping form screenshots for the record?

Reply:
Yup, let me do that as well. There will be a follow up for shipping as well.